vyxalbot2/__init__.py

```diff
@@ -387,7 +387,7 @@ async def runCommand(
                 await self.room.reply(
                     event.message_id, random.choice(self.messages["hugs"])
                 )
-            case "repo-list":
+            case "!repo-list":
```

Comment:
why the `!`s?

Reply:
> why the `!`s?

That was my stsem for fixing #31, but now I realize it doesn't actually work :| Will have to fix.